spec/requests/api/blueprints_spec.rb

```diff
@@ -351,7 +355,10 @@
         :action   => "edit",
         :resource => {
           :bundle => {
-            :service_dialog => nil
+            :service_catalog => {
```

Comment:
@imtayadeway this bug was found when passing both `service_dialog` and `service_catalog`, so it verifies the change that was made.